src/libs/ModifiedExpenseMessage.ts

```diff
@@ -8,6 +8,7 @@ import DateUtils from './DateUtils';
 import getReportPolicyID from './getReportPolicyID';
 import * as Localize from './Localize';
 import * as PolicyUtils from './PolicyUtils';
+import {getReportActionOriginalMessage} from './ReportActionsUtils';
```

Comment:
We always import `ReportActionsUtils` like `import * as ReportActionsUtils`; this would be a precedence. I know it affects line length, but I think we should keep being consistent